histoprint/__init__.py

```diff
@@ -1,5 +1,7 @@
 """histoprint - Pretty-print histograms to the terminal"""
 
-from histoprint.formatter import *
+from histoprint.formatter import HistFormatter, print_hist, text_hist
 
-from .version import version as __version__
+from .version import version as __version__  # noqa: F401
+
+__all__ = ["print_hist", "text_hist", "formatter", "HistFormatter"]
```

Comment:
```suggestion
__all__ = ("print_hist", "text_hist", "formatter", "HistFormatter", "__version__")
```

This does mean that `import *` will import `__version__`, but I think that's expected and one reason users should generally not import star. Also, if you add `def dir`, you can return `__all__` which clearly should contain `__version__`, it's public API.

Reply:
I know that `import *` is bad style, but should we really make it import something starting with `__`? That seems like it could cause unnecessary headaches for (casual) users.